android/runtime/v8/src/native/V8Util.cpp

```diff
@@ -24,7 +24,7 @@ Utf8Value::Utf8Value(v8::Local<v8::Value> value)
   if (value.IsEmpty())
     return;
 
-  v8::Local<v8::String> string = value.As<String>();
+  v8::Local<v8::String> string = value->ToString();
```

Comment:
This variant of ToString is deprecated (and will be removed in V8 6.4), and so is the variant node uses right now that takes the isolate as an argument.

They want you to use the `ToString(Local<Context>)`: https://v8docs.nodesource.com/node-8.0/dc/d0a/classv8_1_1_value.html#a1afa7fc2578fde13194e602513180a65

Unfortunately that isn't easy because we're not passing in a context or the isolate right now.

I believe the easiest way is to actually totally remove our titanium::Utf8Value and titanium::TwoByteValue usage and replace them with v8::String::Utf8Value and v8::String::Value usage (which do exactly what we're attempting to do here, but are now official APIs).